app/controllers/Giraffe.scala

```diff
@@ -84,20 +85,25 @@ class Giraffe(paymentServices: PaymentServices) extends Controller {
   val chargeId = "charge_id"
 
   def maxAmount(currency: Currency): Option[Int] = currency match {
-    case CountryGroup.Australia.currency => 3500.some
+    case Australia.currency => 3500.some
     case _ => 2000.some
   }
 
   def contributeRedirect = NoCacheAction { implicit request =>
-    val countryGroup = request.getFastlyCountry.getOrElse(CountryGroup.RestOfTheWorld)
+
+    val countryGroup = request.getFastlyCountry match {
+      case Some(Canada) | Some(NewZealand) => UK
+      case Some(other) => other
+      case None => RestOfTheWorld
+    }
+
```

Comment:
This change is not strictly necessary, it's just here to avoid redirecting twice. 
We could leave it as it is and it would redirect to /ca or /nz which in turn would redirect to /uk.